src/App.vue

```diff
@@ -166,10 +169,30 @@ import Alerter from './components/widgets/Alerter.vue'
 import { datalogger } from './libs/sensors-logging'
 import { useMainVehicleStore } from './stores/mainVehicle'
 import { useWidgetManagerStore } from './stores/widgetManager'
-
 const widgetStore = useWidgetManagerStore()
 const vehicleStore = useMainVehicleStore()
 
+onMounted(async () => {
```

Comment:
this function also probably need to do the same with the RTC connection and maybe the base URL